utils/posts.js

```diff
@@ -43,7 +43,13 @@ export function getPostMetadata(id) {
     };
 
     const title = post.meta.title;
-    const seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, '-');
+    var seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, '-');
```

Comment:
This is interesting. This code probably works, but it's far too many lines of code to accomplish the little thing we want to do.

I'm wondering if we could optimize better by updating the regex or if there are some flags that we could pass in to `.replace(...)` ?